### DISCUSSION TITLE
How can I configure Nginx Proxy manager to use a Private CA for SSL?

### DISCUSSION THREAD

**Discussion by ThatNerdUKnow** [NginxProxyManager/nginx-proxy-manager#2180] [Q&A]
Hi All, I'm working on securing my home network and this is what my stack looks like
`bind9` - dns server
`step-ca` - certificate authority, configured as an ACME server
`nginx-proxy-manager` - http proxy
`unraid` - host os

I currently set up a private domain name on bind9, and I want to generate a wildcard certificate for this domain via the ssl feature of nginx proxy manager. How can I get it to work with my `step-ca` installation?

**Comment by ThatNerdUKnow** [comment]
I tried to use the `ACME-DNS` challenge but I get the following command output
```
Error: Command failed: certbot certonly --config "/etc/letsencrypt.ini" --cert-name "npm-4" --agree-tos --email "brandonpi265@gmail.com" --domains "*.dogma.net" --authenticator dns-acmedns --dns-acmedns-credentials "/etc/letsencrypt/credentials/credentials-4"
Saving debug log to /var/log/letsencrypt/letsencrypt.log
File not found: /data/acme-registration.json
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.

    at ChildProcess.exithandler (node:child_process:399:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

It seems that I need to write a configuration file at /data/acme-registration.json
But I can't seem to find any documentation on how to configure this file

**Reply by mfide** [reply]
could you solve it?

**Reply by ThatNerdUKnow** [reply]
> could you solve it?

No
